src/nft/nft.controller.ts

```diff
@@ -8,57 +8,63 @@ import {
   UploadedFile,
   UseGuards,
   UseInterceptors,
-} from '@nestjs/common';
-import { FileInterceptor } from '@nestjs/platform-express';
-import { User } from '@prisma/client';
-import { log } from 'console';
-import { CurrentUser } from 'src/auth/decorators/current.user.decorator';
-import { AuthGuard } from 'src/auth/guards/auth.guard';
-import { StorageService } from 'src/storage/storage.service';
-import { NftService } from './nft.service';
+} from "@nestjs/common";
+import { FileInterceptor } from "@nestjs/platform-express";
+import { User } from "@prisma/client";
+import { CurrentUser } from "src/auth/decorators/current.user.decorator";
+import { AuthGuard } from "src/auth/guards/auth.guard";
+import { StorageService } from "src/storage/storage.service";
+import { NftService } from "./nft.service";
+import { UploadDto } from "./dto/upload.dto";
 
 @UseGuards(AuthGuard)
-@Controller('nft')
+@Controller("nft")
 export class NftController {
   constructor(
     private storageService: StorageService,
     private nftService: NftService,
   ) {}
 
-  @Get('/user')
+  @Get("/user")
```

Comment:
There's no need to use `/user` path, sending request to `/nfts` is supposed to return only nfts that belong to current user.